test/libsolidity/util/TestFunctionCall.h

```diff
@@ -93,6 +101,7 @@ class TestFunctionCall
 	void setRawBytes(const bytes _rawBytes) { m_rawBytes = _rawBytes; }
 	void setGasCost(std::string const& _runType, u256 const& _gasCost) { m_gasCosts[_runType] = _gasCost; }
 	void setContractABI(Json::Value _contractABI) { m_contractABI = std::move(_contractABI); }
+	void setActualSideEffects(std::vector<std::string> _sideEffects) { m_call.actualSideEffects = _sideEffects; }
```

Comment:
Both `setGasCost()` and `setActualSideEffects()` set actual values so I think they should be named consistently. Either `setActualGasCost()` + `setActualSideEffects()` or `setGasCost()` + `setSideEffects()`.

Also, since these mechanisms are similar, maybe `m_gasCosts` and the side effects should be stored in the same place as well?